lexicons/tools/ozone/set/removeSet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.removeSet",
```

Comment:
I'd recommending having an `output` from this lexicon, even if it's empty for now.  It helps with future-proofing, since it's sort of a breaking change to transition from no response to some response.

Reply:
this is a great point for a general lexicon style guide, I would not have thought of this.